docs/operator-manual/notifications/troubleshooting-errors.md

```diff
@@ -39,3 +39,7 @@ You need to check your argocd-notifications controller version. For instance, th
 ### notification service 'xxxx' is not supported"
 
 You have not defined `xxxx` in `argocd-notifications-cm` or to fail to parse settings.
+
+### channel_not_found
```

Comment:
I think it is slack specific issue, maybe better put it somewhere in slack documentation instead generic one?